pkg/apis/ceph.rook.io/v1/types.go

```diff
@@ -1770,6 +1770,21 @@ type GatewaySpec struct {
 	// +nullable
 	// +optional
 	RgwCommandFlags map[string]string `json:"rgwCommandFlags,omitempty"`
+
+	// RgwReadAffinity defines the read affinity policy to optimize the read requests for the RGW clients
+	// Note: Only supported from Ceph Tentacle (v20)
+	// +optional
+	RgwReadAffinity *RgwReadAffinity `json:"rgwReadAffinity,omitempty"`
```

Comment:
IMO no need to add the RGW prefix
```suggestion
	ReadAffinity *RgwReadAffinity `json:"rgwReadAffinity,omitempty"`
```